deployments/puppet/manifests/init.pp

```diff
@@ -32,6 +32,7 @@
   $fluentd_config_dest     = $splunk_otel_collector::params::fluentd_config_dest,
   $fluentd_capng_c_version = '<=0.2.2',  # linux only
   $fluentd_systemd_version = '<=1.0.2',  # linux only
+  $gomemlimit              = '460',
```

Comment:
1. Should the default be `''` instead?
2. Please re-add this to the [template](https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/puppet/templates/splunk-otel-collector.conf.erb) for the linux service if the value is non-empty.
3. Please document this new parameter in the [readme](https://github.com/signalfx/splunk-otel-collector/blob/main/deployments/puppet/README.md).

Reply:
We should not hardcode GOMEMLIMIT by default.